plugins/module_utils/aci.py

```diff
@@ -1772,3 +1772,10 @@ def api_call(self, method, url, data=None, return_response=False):
                 except KeyError:
                     # Connection error
                     self.fail_json(msg="Connection failed for {url}. {msg}".format_map(info))
+
+    def delete_child(self, path):
```

Comment:
why call this function delete child?
why the wrapper around, can't you do this directly from the module with a single?
why not use the self.base_url, which is set during constructor of the ACIModule?